tests/integration/test_fim/test_files/test_whodata_policy_change/test_whodata_policy_change.py

```diff
@@ -0,0 +1,174 @@
+    - fim_mode:
+        whodata: Implies real-time monitoring but adding the 'who-data' information.
+    - tier:
+        0: Only level 0 tests are performed, they check basic functionalities and are quick to perform.
+        1: Only level 1 tests are performed, they check functionalities of medium complexity.
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_whodata_policy_change
+'''
+import os
+import time
+
+import pytest
+from wazuh_testing.tools import PREFIX, configuration
+from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.tools.local_actions import run_local_command_returning_output
+from wazuh_testing import global_parameters, LOG_FILE_PATH
+from wazuh_testing.modules import fim
+from wazuh_testing.modules.fim import event_monitor as evm
+from wazuh_testing.modules.fim import FIM_DEFAULT_LOCAL_INTERNAL_OPTIONS as local_internal_options
+from wazuh_testing.modules.fim.utils import regular_file_cud
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
+
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+test_cases_path = os.path.join(TEST_CASES_PATH, 'cases_whodata_policy_change.yaml')
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_whodata_policy_change.yaml')
+
+
+# variables
+test_folders = [os.path.join(PREFIX, fim.TEST_DIR_1)]
+folder = test_folders[0]
+file_list = [f"regular_file"]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Test configurations
+configuration_parameters, configuration_metadata, test_case_ids = configuration.get_test_cases_data(test_cases_path)
+for count, value in enumerate(configuration_parameters):
+    configuration_parameters[count]['TEST_DIRECTORIES'] = folder
+configurations = configuration.load_configuration_template(configurations_path, configuration_parameters,
+                                                           configuration_metadata)
+
+
+# tests
+@pytest.mark.parametrize('test_folders', [test_folders], ids='', scope='module')
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=test_case_ids)
+def test_whodata_policy_change(configuration, metadata, set_wazuh_configuration, create_monitored_folders_module,
+                               configure_local_internal_options_function, restart_syscheck_function,
+                               wait_fim_start_function):
+    '''
+    description: Check if the 'wazuh-syscheckd' is monitoring a in whodata mode in Windows, and the Audit Policies are
+                 changed, the monitoring changes to realtime and works on the monitored files.
+
+    wazuh_min_version: 4.5.0
+
+    tier: 1
+
```

Comment:
Add `setup`, `test` and `tierdown` documentation sections

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3627/commits/0d431cc2ea9340d5f7d0aef41bc14acd57a26ff0